flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "automerge-flathubbot-prs": true
+}
```

Comment:
I don't think we should merge changes automatically but that might just be me being cautious.

Reply:
I know its contentious but its just a suggestion for @wberrier

Reply:
I'll agree with @michaelkuhn , this opens the door to publishing a broken/buggy makemkv for users.  I'd be more comfortable if we had some basic sanity checks.